tests/sharingTest.js

```diff
@@ -1,33 +1,41 @@
 import { MatchersV3, XmlBuilder } from '@pact-foundation/pact/v3'
 
+const { fromProviderState } = MatchersV3
+const { toFormUrlEncoded } = require('./helpers/sharingHelper')
+
 describe('Main: Currently testing file/folder sharing,', function () {
   const config = require('./config/config.json')
-  const username = config.adminUsername
 
   const {
     applicationXmlResponseHeaders,
     applicationFormUrlEncoded,
-    accessControlAllowHeaders,
-    accessControlAllowMethods,
-    validAdminAuthHeaders,
-    xmlResponseHeaders,
+    htmlResponseHeaders,
     getCapabilitiesInteraction,
     getCurrentUserInformationInteraction,
     createOwncloud,
     ocsMeta,
     shareResponseOcsData,
     createProvider,
-    origin
+    getAuthHeaders
   } = require('./pactHelper.js')
 
   // TESTING CONFIGS
   const testUserPassword = config.testUserPassword
-  const testUser = config.testUser
   const testGroup = config.testGroup
-  const testFolder = '/' + config.testFolder
+  const testFolder = config.testFolder
   const nonExistentFile = config.nonExistentFile
   const expirationDate = config.expirationDate
   const testFiles = config.testFiles
+  const testFilesId = config.testFilesId
+
+  const sharer = config.testUser
+  const sharerPassword = config.testUserPassword
+  const sharee = config.testUser2
+  const shareePassword = config.testUser2Password
+
+  const validAuthHeaders = {
+    authorization: getAuthHeaders(sharer, sharerPassword)
+  }
```

Comment:
IMO, this can be used in other files as well, so we can keep this in pact helper directly, with getAuthHeaders, validAdminAuthHeaders. Using it this way can be uniform when we can use in other files as well.

Reply:
we can make it at once for all tests in another PR